docs/migration-guides/1.10-1.11.md

```diff
@@ -131,4 +141,58 @@ Perform the following steps to see if the migration process was successful.
 
 After the migration, use the new Knative Eventing Mesh endpoint, `https://gateway.domain/APP_NAME/events`, to send events.
 
-**NOTE**: The old endpoint, `https://gateway.domain/APP_NAME/v1/events`, will be handled by the compatibility layer which uses the new Eventing Mesh to route events from the old endpoints to the sinks.
\ No newline at end of file
+**NOTE**: The old endpoint, `https://gateway.domain/APP_NAME/v1/events`, will be handled by the compatibility layer which uses the new Eventing Mesh to route events from the old endpoints to the sinks.
+
+### Verify event processing
+
+After the upgrade, event processing may encounter some issues caused by Subscriptions not being removed and recreated properly. To fix this, perform the following steps:
+
+1. Get all the Triggers in the cluster:
+
+```bash
+kubectl get triggers.eventing.knative.dev -A
+```
+
+If an issue occurred, Triggers will have the **READY** parameter set to `False`. See the example:
+
+```bash
+NAMESPACE   NAME                                   READY   REASON                 BROKER    SUBSCRIBER_URI           AGE
+test        654331ad-f6ab-56ab-bd91-e0b397afb7b8   False   NotSubscribed          default   http://test.test:8080/   39h
+```
+
+3. Get Trigger details:
+
+```bash
+kubectl describe triggers.eventing.knative.dev -n {namespace} {trigger_name}
+```
+4. If the status of the Trigger is `Type:Ready Status:False` with a message `object is being deleted: subscriptions.messaging.knative.dev`, get the Subscriptions for the Triggers:
+
+```bash
+kubectl get subscriptions.messaging.knative.dev -n {namespace} -l  eventing.knative.dev/trigger=<trigger_name>
+```
+
+5. Scale down `eventing-controller`:
+
+```bash
+kubectl scale deploy -n knative-eventing eventing-controller --replicas=0
+```
+
+6. Patch the Subscriptions:
+
+```bash
+kubectl patch -n <namespace> subscriptions.messaging.knative.dev {subscription_name} --type merge -p '{"metadata": {"finalizers": []}}'
+```
+
+7. Scale up `eventing-controller`:
+
+```bash
+kubectl scale deploy -n knative-eventing eventing-controller --replicas=1
+```
+
+Deleting the finalizer makes the garbage collector remove the old Subscriptions and triggers the creation of new Subscriptions. This way, Triggers can again be linked to Subscriptions and ready to use.
```

Comment:
```suggestion
Deleting the finalizer makes the garbage collector remove the existing Subscriptions and triggers the creation of new Subscriptions. This way, you can link Triggers to Subscriptions again and use them.
```

Reply:
But actually... is this part needed: `and use them.`? If yes, maybe we could explain how we can use them now.

Reply:
No, but I would leave the passive voice here because it's not the user who links the Triggers to Subscriptions in this case.  I changed the wording to improve it.